miniscope_io/models/stream.py

```diff
@@ -174,6 +178,7 @@ class StreamDevConfig(MiniscopeConfig, YAMLMixin):
     reverse_header_bytes: bool = False
     reverse_payload_bits: bool = False
     reverse_payload_bytes: bool = False
+    dummy_words: int = 0
```

Comment:
Yes, it's just increasing because I'm adding whatever I need to harness the hardware part. I think this will keep increasing, but at least the categories are getting fixed, so yeah, refactoring makes sense.

I might be too used to wireless communication and packet studies to notice the missing description we'll need for the buffer, so let me know if you notice something. There are some skipped (not fixed) descriptions of metadata now, but if there is other stuff.